pontoon/base/views.py

```diff
@@ -264,11 +264,17 @@ def contributor(request, email):
         })
 
     timeline.reverse()
+    words = []
```

Comment:
So do I just use a counter and increment `+=` or do I do what is being done here and return only the  `len(words)` to the view instead of doing `{{ words | length }}` ?

I used it this way because I felt it might come in handy since translations from the QuerySet is also being sent to the view where the `.count()` is calculating the values.

Should I be changing this @jotes :smile: